scripts/index.js

```diff
@@ -8,4 +8,6 @@ const DEFAULT_SERVER_URL = "https://kinto.dev.mozaws.net/v1";
 
 const kinto = new Kinto({remote: DEFAULT_SERVER_URL});
 const store = new Store(kinto);
+store.load();
```

Comment:
How about calling this when App is mounted? as a bonus we could then write a test for it.